AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -1161,12 +1166,13 @@ - (void)trackModelException:(MSException *)exception withProperties:(NSDictionar
   if (properties && properties.count > 0) {
 
     // Send only valid properties.
-    log.properties =
-        [MSUtility validateProperties:properties forLogName:[NSString stringWithFormat:@"ErrorLog: %@", log.errorId] type:log.type];
+    log.properties = [MSUtility validateProperties:properties
+                                        forLogName:[NSString stringWithFormat:@"ErrorLog: %@", log.errorId]
+                                              type:log.type];
   }
 
   // Enqueue log.
-  [self.channelUnit enqueueItem:log];
+  [self.channelUnit enqueueItem:log flags:MSFlagsNone];
```

Comment:
Should it be `Normal`?

Reply:
None needs to map to default values for each flags as discussed.

Reply:
Not sure about log lines but crashes should be critical while attachments and handled errors are normal.